### PR TITLE
express api initial

### PR DESCRIPTION
**Pull request by BenAAndrew**

### CODE REVIEW

**Comment by BenAAndrew** [comment]
@lynneagletonkainos could you test please :)

**Comment by weesteeky** [comment]
Looks good :)

**Review by weesteeky — APPROVED** [review]

File: express.js [hunk 1]
```diff
@@ -1,42 +1,47 @@
...
+    var tableName = 'Course';
+    var queryName = 'INSERT INTO ' + tableName + '(Title, Date, Location, Description, Owner, TargetAudience) \nVALUES(' + '"';
+
+    queryName = queryName + + '"'+ data[0]+'"' + ', ';
+    queryName = queryName +'"'+ data[1]+'"' + ', ';
+    queryName = queryName +'"'+ data[2] +'"'+ ', ';
+    queryName = queryName +'"'+ data[3]+'"' + ', ';
+    queryName = queryName +'"'+ data[4] +'"'+ ', ';
+    queryName = queryName +'"'+ data[5]+'"';
+    //queryName = queryName + dummyData[6] + ');';
```
Comment (ghost) [inline express.js]: This should be removed completely if not needed. You can check back through commit history via GitHub if you need to see old code.

Comment (BenAAndrew) [reply express.js]: @lynneagletonkainos Good point will remove it now. We've actually completely re structured this function in another branch which we'll merge in soon hopefully :)

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**